capstone/fabfile.py

```diff
@@ -205,7 +206,7 @@ def add_test_case(*barcodes):
     from django.core import serializers
     from tracking_tool.models import Volumes, Reporters, BookRequests, Pstep, Eventloggers, Hollis, Users
     from capdb.storages import ingest_storage
-    from scripts.helpers import parse_xml, copy_file, resolve_namespace, serialize_xml
+    from scripts.helpers import copy_file, resolve_namespace
 
```

Comment:
seems like there are two import lines from scripts.helpers
Want to merge them?

Reply:
Nice catch, thanks!